src/pages/governance/_components/VotingResult.tsx

```diff
@@ -91,91 +91,77 @@ export function VotingResult({
               )}
             >
               <div className="flex flex-col gap-y-2 w-full">
-                <span className="md:text-lg text-sm font-semibold text-gray-900 dark:text-dark-gray-900">
-                  Yes
-                </span>
-                <Progress
-                  value={percYes.toNumber()}
-                  containerClass="bg-gray-100 dark:bg-dark-gray-200"
-                  contentClass="bg-gray-300 dark:bg-dark-gray-400"
-                />
                 <div className="flex flex-row">
-                  <span
-                    className={classNames(
-                      "grow text-gray-600 dark:text-dark-gray-600 text-sm"
-                    )}
-                  >
-                    {percYes.toFixed(2)}%
+                  <span className="md:text-base text-sm font-semibold text-gray-900 dark:text-dark-gray-900 grow">
+                    Yes
+                  </span>
+                  <span className="md:text-base text-sm font-semibold text-gray-900 dark:text-dark-gray-900">
+                    No
                   </span>
-                  <NumericFormat
-                    value={voteCounts.yes}
-                    fixedDecimalScale
-                    thousandSeparator=","
-                    displayType="text"
-                    className="text-sm text-gray-600 dark:text-dark-gray-600"
-                  />
                 </div>
-              </div>
 
-              <div className="flex flex-col gap-y-2 w-full mt-4">
-                <span className="md:text-lg text-sm font-semibold text-gray-900 dark:text-dark-gray-900">
-                  No
-                </span>
                 <Progress
-                  value={percNo.toNumber()}
+                  yesValue={percYes.toNumber()}
+                  noValue={percNo.toNumber()}
+                  approvalThreshold={Number(
+                    proposal.approvalThreshold.replace("%", "")
+                  )}
                   containerClass="bg-gray-100 dark:bg-dark-gray-200"
-                  contentClass="bg-gray-300 dark:bg-dark-gray-400"
+                  // contentClass="bg-gray-300 dark:bg-dark-gray-400"
```

Comment:
we need this?

Reply:
ahh i caught this just now going to push